tests/FsToolkit.ErrorHandling.Tests/SeqPerformanceTests.fs

```diff
@@ -0,0 +1,42 @@
+module SeqPerformanceTests
+
+#if !FABLE_COMPILER
+open Expecto
+open Expecto.Tests
+open Expecto.Performance
+open FsToolkit.ErrorHandling
+open SampleDomain
+
+let inline repeat n a arg =
+    [for _ in 0..(n - 1) -> a arg]
+
+let inline repeat10 a arg =
+    repeat 10 a arg
+
+let traverseResultTests =
+    testSequenced <| testList "Seq performance tests " [
```

Comment:
I thought these tests will show that converting `Array -> Seq -> (Traversable operation against the seq) -> Array` will be much faster than `Array -> List -> (Traversable opreation against the List) -> Array` .

But interestingly the result was actually opposite. Latter was about 60x faster.